cfme/tests/v2v/test_csv_import.py

```diff
@@ -76,13 +81,24 @@ def import_and_check(appliance, infra_map, error_text=None,
         else:
             # click on the checkbox to select VM (column 0)
             plan_view.vms.table[0][1].widget.click()
-        error_msg = plan_view.vms.popover_text.read()
+        if not security_group:
+            error_msg = plan_view.vms.popover_text.read()
     else:
         if alert:
             error_msg = plan_view.browser.get_alert().text
-            plan_view.browser.handle_alert()
+            try:
+                plan_view.browser.handle_alert()
+            except NoSuchElementException:
+                pass
         else:
             error_msg = plan_view.vms.error_text.text
+    if security_group:
+        plan_view.next_btn.click()
+        plan_view.instance_properties.table.wait_displayed()
+        table_data = plan_view.instance_properties.table.read()[0]
+        error_msg = {
+            "security_group": table_data["OpenStack Security Group"],
+            "flavor": table_data["OpenStack Flavor"]}
     plan_view.cancel_btn.click()
     return bool(error_msg == error_text)
```

Comment:
Again, general comment not specific to this PR, but something to consider as you work on these test modules.

These variables name/use is misleading, as `error_text` is expected to actually be a dictionary with very specific keys, not `text` as the name implies. Same thing with `error_msg`.  There's no docblock for the function explaining what its actually doing (or in-line comments explaining what is happening).

Additionally the test is making direct references to table columns after having read the dictionary, without asserting/checking that those columns are present. 

Additionally this would be a much more useful function if it was actually using deepdiff to compare the dictionaries. This will result in a diff that actually describes the differences between the dicts.  The assertion being here would also help, as then the test isn't asserting the return of some function, but the actual data.

Reply:
There are few things..

>These variables name/use is misleading, as error_text is expected to actually be a dictionary with very specific keys, not text as the name implies. Same thing with error_msg. 

Basically only in `test_csv_security_group_flavor` argument is dictionary, in this case I was using same `import_or_check` function so I used existing variables which I accepts misleading since it is not text as you noticed but I intentionally added that since it was exceptional.

>There's no docblock for the function explaining what its actually doing (or in-line comments explaining what is happening).

Let me update it.

>Additionally this would be a much more useful function if it was actually using deepdiff to compare the dictionaries. This will result in a diff that actually describes the differences between the dicts. The assertion being here would also help, as then the test isn't asserting the return of some function, but the actual data.

Got you, since it is only with one function I like to go with what I have but I totally agree and I will make sure I will use deepdiff when I compare dictionary from next time.

Reply:
Updated functions docstring and used deepdiff.

Reply:
@mshriver 5.11 was failing because deepdiff was comparing unicode string (value after reading table) with normal string (value of expected attributes) which was not equal, so I [added](https://github.com/ManageIQ/integration_tests/pull/9016/commits/776fa4628104d9e6021fa9e1f2f71c8f9465acbc) `str` conversion around values we are getting from table.

I could have just used `==` comparison too but I am preferring deepdiff right now.